reference/commands/cache.rst

```diff
@@ -146,7 +146,8 @@ conan cache clean
 
 This command will remove all temporary folders, along with the source, build and download folder
 that Conan generates in its execution. It will do so for every matching reference passed in *pattern*,
-unless a specific flag is supplied, in which case only the specified folders will be removed.
+or the contents of the pkglist file if the ``--list`` option is used.
+It's possible to limit the cleaning to certain kinds of folders with different flags.
```

Comment:
May we add an example of the use down below?

Reply:
Done